include/ensmallen_bits/pso/pso.hpp

```diff
@@ -145,8 +149,8 @@ class PSOType
               VelocityUpdatePolicy(),
           const InitPolicy& initPolicy = InitPolicy()) :
           numParticles(numParticles),
-          lowerBound(lowerBound * arma::ones(1, 1)),
-          upperBound(upperBound * arma::ones(1, 1)),
+          lowerBound(lowerBound * MatType(1, 1, GetFillType<MatType>::ones)),
+          upperBound(upperBound * MatType(1, 1, GetFillType<MatType>::ones)),
```

Comment:
I saw it somewhere else too and commented there also, but I think `lowerBound({ lowerBound })` will work here.